API.yaml

```diff
@@ -26,12 +26,14 @@ info:
   license:
     name: Apache 2
     url: http://www.apache.org/licenses/
-  version: 0.2.0
+  version: 0.3.0
 servers:
 - url: https://localhost:8080/tsp/api
 tags:
 - name: Annotations
   description: Retrieve annotations for different outputs.
+- name: Bookmarks
```

Comment:
I think you added it manually and it's generated by the server definitions. See comment here: https://github.com/eclipse-tracecompass-incubator/org.eclipse.tracecompass.incubator/pull/102#pullrequestreview-245536650

Please update the server PR for that.